hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/ContractsV_0_32Module.java

```diff
@@ -64,7 +65,12 @@ public interface ContractsV_0_32Module {
     @ContractsModule.V_0_32
```

Comment:
General change request, we should update the `V_0_32` and all related classes to the version we expect this to ship in, which I'm fairly certain is no longer 32.

Reply:
Will resolve in a future PR